var/spack/repos/builtin/packages/py-amrex/package.py

```diff
@@ -64,15 +76,23 @@ class PyAmrex(PythonPackage, CudaPackage, ROCmPackage):
         depends_on("amrex +rocm")
         # todo: how to forward amdgpu_target?
 
+    depends_on("py-pytest", type="test")
+    depends_on("py-pandas", type="test")
+    depends_on("py-cupy", type="test", when="+cuda")
+
+    tests_src_dir = "tests/"
+
     def setup_build_environment(self, env):
         spec = self.spec
 
         # disable superbuilds: use external dependencies
         env.set("AMREX_INTERNAL", "OFF")
+        env.set("PYAMREX_CCACHE", "ON")
```

Comment:
Does this need `ccache` as a build dependency?

Reply:
Oh no, that should be
```suggestion
        env.set("PYAMREX_CCACHE", "OFF")
```

Reply:
#43570